ownCloud/AppDelegate.swift

```diff
@@ -73,6 +73,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 		// Licenses
 		OCExtensionManager.shared.addExtension(OCExtension.license(withIdentifier: "license.libzip", bundleOf: Theme.self, title: "libzip", resourceName: "libzip", fileExtension: "LICENSE"))
 
+		//Disable UI Animation for UITesting (screenshots)
```

Comment:
Is it possible to move this to a test class only, if we do no need this for production?

Reply:
It have to be inside the app code. Because of that I added a new env variable to validate if it is necessary disable it or not:
`app.launchEnvironment = ["oc:app.show-beta-warning": "false", "oc:app.enable-ui-animations": "false"]`

By default it is true:
`.enableUIAnimations: true`